composer.lock

```diff
@@ -9290,17 +9290,17 @@
         },
         {
             "name": "drupal/scheduled_transitions",
-            "version": "2.2.1",
+            "version": "2.2.0",
```

Comment:
We dont want to downgrade this. We need the patch release.

Reply:
> Sorry, I meant "testing steps" like steps for manually testing it. But an automated test is even better. Lets call upon @deviantintegral and @tom-fleming for input on mocking time service for functional tests like those used on this project. I see https://www.drupal.org/project/datetime_testing which looks like a more polished version of what you have done with State.

No problem, Moshe. The `datetime_testing` module looks promising, I can try that for sure, but I'm interested in @tom-fleming  and @deviantintegral's opinion. Thanks!

Reply:
Nice find! I only looked at a few files, but they look good. Multiple independent implementations [reaching the same conclusion](https://git.drupalcode.org/project/datetime_testing/-/blob/8.x-1.x/src/TestTime.php#L72-81) is nice to see.